pymeasure/display/widgets.py

```diff
@@ -233,6 +254,15 @@ def update_y_column(self, index):
         axis = self.columns_y.itemText(index)
         self.plot_frame.change_y_axis(axis)
 
+    def load(self, curve):
+        self.plot.addItem(curve)
+
+    def remove(self, curve):
+        self.plot.removeItem(curve)
+
+    def set_color(self, curve, color):
+        """ Remove curve from widget """
+        curve.setPen(pg.mkPen(color=color, width=2))
 
 class ImageFrame(QtGui.QFrame):
```

Comment:
It seems there is quite a bit of overlap with the `PlotFrame` class; would it be possible to inherit from that class?

Reply:
Indeed you are right. I made the changes in my commit